js/moveset_import.js

```diff
@@ -262,7 +260,12 @@ function addSets(pokes) {
 					currentPoke.nameProp = "Custom Set";
 				}
 				currentPoke.isCustomSet = true;
-				currentPoke.ability = getAbility(rows[i + 1].split(":"));
+				if (currentPoke.name.indexOf("ite") === -1) {
```

Comment:
Err, either `currentPoke.name` is named very badly, or this is wrong and you should be testing item name instead.

Reply:
Yeah this should be ``currentPoke.item`` but it’s not working as intended so I’m brainstorming a bit more.